t/26-video_stream.t

```diff
@@ -94,6 +98,33 @@ subtest 'connect stream' => sub {
         [('/dev/video0', '--get-dv-timings')],
     ], "calls to v4l2-ctl";
 
+    @v4l2_ctl_calls = ();
+    $console->connect_remote({url => 'ustreamer:///dev/video0'});
+    is $console->{dv_timings_supported}, 0, "correctly skipping DV timing";
+    is_deeply \@v4l2_ctl_calls, [], "calls to v4l2-ctl";
+
+    my $cmd = $mock_console->original('_get_ustreamer_cmd')->($console, '/dev/video0', 'raw-sink-dev-video0');
+    is_deeply $cmd, [
+        'ustreamer', '--device', '/dev/video0', '-f', '5',
+        '-c', 'NOOP',
+        '--raw-sink', 'raw-sink-dev-video0', '--raw-sink-rm',
+        '--dv-timings'], "correct cmd built";
+
+};
+
+subtest 'long double encoding' => sub {
+    my $console = consoles::video_stream->new(undef, {url => 'ustreamer:///dev/video0'});
+    my $value = 5746059.6457196;
+    my $expected;
+    eval {
+        $expected = pack("D", $value);
+    };
+    if ($@) {
+        ok 1, "skip, nothing to compare to";
+    } else {
+        my $actual = $console->_pack_long_double($value);
+        is $actual, $expected, "correctly encoded";
+    }
```

Comment:
IMHO this if/else in a test does not make sense. Tests on the top level should be designed to be reproducibly executed all the time so this should be changed.

Reply:
Should I hardcode "correct" encoding of example value then? The point of this test is testing if it gets the same result as `pack("D", ...)` (but for that I need a perl version that supports that)

Reply:
Uhm, this actually seems to be broken on arm64...

Can I somehow declare that ustreamer is not supported on openSUSE Leap and use `pack("D")`?

Reply:
I guess that should be fine

Reply:
How to do that, more specifically how to skip those tests on too old Perl? And where to document this limitation?

Reply:
Well, I wouldn't skip tests based on a specific version. Better to only conduct those tests that matter for the actual implementation in os-autoinst. So how about leaving out that specific test for the value encoding?

Reply:
The thing is, if I use `pack("D")`, then all ustreamer tests will fail on the older Perl version

Reply:
To be clear: I'd like to simply drop [the last commit](https://github.com/os-autoinst/os-autoinst/pull/2417/commits/333a02671cbcb59e3564de1752592b0b8e3826fd). And then do something to make it not complain on the old Perl version (accepting it will not work there).